src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super("Circle", color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void displayInfo() {
+        System.out.println("Figure: circle"
+                + ", area: " + getArea() + " sq.units"
+                + ", radius: " + radius
+                + ", color: " + super.getColor());
```

Comment:
```suggestion
                + ", color: " + getColor());
```

в методах `getArea()` и `displayInfo()` не нужно использовать ключевые слова `super` `this`. Проверь все класы фигур, пожалуйста.